site/src/pages/components/interest-invokers.explainer.mdx

```diff
@@ -95,15 +104,15 @@ For the above reasons, a somewhat more complicated approach is adopted for `inte
 - if the target is a popover, and the popover contains keyboard-focusable elements, then it put into "partial interest" mode, whereby interactive contents within the popover act as if they have `tabindex=-1`. This keeps the popover from inserting any additional tab stops into the sequential focus navigation order.
 - a hotkey (e.g. Alt-UpArrow) will cause the popover to be "fully activated" which removes the special `tabindex=-1` behavior, making it available in the focus navigation order. This hot-key must be chosen by the UA to be convenient for the user, while also not conflicting with existing UA-provided and OS-provided hot-keys. Other ideas include Alt-Space or Ctrl-Space for the show interest hot key. The hotkey should not be an arrow key *without* a modifier, since that will interfere with scrolling and editing.
-- a new set of pseudo classes will match on the interest invoker *and* the target popover, only when it is in the "partially activated" state. For example, `:has-partial-interest` will match the invoker, and `:target-of-partial-interest` will match the popover.
+- a new set of pseudo classes will match on the interest invoker *and* the target popover, only when it is in the "partial interest" state. For example, `:has-partial-interest` will match the invoker, and `:target-of-partial-interest` will match the popover.
 - the special `tabindex=-1` behavior will be implemented with a new UA stylesheet rule: `:target-of-partial-interest {interactivity:not-keyboard-focusable}`.  See the [interactivity:not-keyboard-focusable section](#interactivity-not-keyboard-focusable) for more detail.
-- a new UA stylesheet rule will will also be added, such as: `:target-of-partial-interest::after {content: "Press Alt-UpArrow to activate this content"}`. This adds (developer-stylable or -hideable) hints to the user about the hotkey, so that it is discoverable.
+- a new UA stylesheet rule will will also be added, such as: `:target-of-partial-interest::after {content: "Press Alt-UpArrow to activate"}`. This adds (developer-stylable or -hideable) hints to the user about the hotkey, so that it is discoverable.
 
 This approach, while slightly more complicated, nicely meets the following use case requirements:
- - Better user activation story for non-interactive "tooltips" that shouldn't be so hard to activate. Note that these have no behavior difference between "partially activated" and "fully activated", since they contain nothing interactive.
+ - Better user activation story for non-interactive "tooltips" that shouldn't be so hard to activate. Note that these have no behavior difference between "partial interest" and "full interest", since they contain nothing interactive.
  - Better discoverability story for rich "hovercards". The `::after` UA rule provides explicit instructions.
```

Comment:
I know it was discussed - but it doesn't have to be solved in this PR - we should avoid populating `::after` and have a separate pseudo element I think.

Reply:
Yeah, agreed. I've put it in `::after` for now, just to avoid the "work" of creating that pseudo. If this whole partial interest thing sticks, then I agree and I'll open an issue. (Some a11y folks are doing user studies on whether "partial interest" is actually good for users, or whether it's just better to show full interest on focus.)